packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/FileUtils.java

```diff
@@ -215,4 +215,8 @@ private static boolean isMediaDocument(Uri uri) {
   private static boolean isGooglePhotosUri(Uri uri) {
     return "com.google.android.apps.photos.contentprovider".equals(uri.getAuthority());
   }
+
+  private static boolean isOneDriveUri(Uri uri) {
+    return "com.microsoft.skydrive.content.external".equals(uri.getAuthority());
```

Comment:
Is there a document I can check to see this is correct?

Reply:
Unfortunately, I can't find it in any official documentation. Got it in an experimental way when selecting photos from OneDrive provider with uri.getAuthority()